configure.ac

```diff
@@ -1972,6 +1972,7 @@ CONFIG_FILES_EXEC([cts/cts-cli],
                   [cts/support/fence_dummy],
                   [cts/support/pacemaker-cts-dummyd],
                   [daemons/fenced/fence_legacy],
+                  [daemons/fenced/fence_watchdog],
```

Comment:
This needs to go in gitignore as well

Reply:
oh crap - yes ... no file-type to be caught by one of the wildcards ...
done